tock/employees/models.py

```diff
@@ -180,6 +179,15 @@ def is_late(self):
         else:
             return False
 
+    def display_name(self):
+        user = self.user
+        if user.first_name and user.last_name:
+            return user.first_name + " " + user.last_name
+        elif user.first_name:
+            return user.first_name
+        else:
+            return user.username   
+
```

Comment:
@Sgtpluck Django has an existing `get_full_name` method on the User object, so we could just sorta alias that: https://docs.djangoproject.com/en/3.1/ref/contrib/auth/#django.contrib.auth.models.User.get_full_name

Also, since it does entail a DB lookup for something that doesn't change often, Django's cached properties might make sense here: https://medium.com/@fdemmer/django-cached-property-on-models-f4673de33990

Reply:
Resolved by updates